src/server/controllers/user.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
```

Comment:
You don't need this: http://stackoverflow.com/questions/31685262/not-recommended-to-write-out-use-strict-with-es6